portal/common/lib/rpc_selector.ts

```diff
@@ -44,12 +43,25 @@ class RPCSelector implements RPCSelectorInterface {
         this.selectedClient = undefined;
     }
 
+    /**
+     * Get the list of RPC URLs from the environment variable.
+     */
+    private static parseRPCList(): string[] {
+        const rpcList = process.env.RPC_URL_LIST;
+        if (!rpcList) {
+            throw new Error("No RPC list found in environment variables");
+        }
+        return rpcList.split(",");
```

Comment:
Question: how is the list provided? as comma-separated i assume?

Reply:
Yes the list is provided as comma separated. e.g.
```
RPC_LIST=https://node1.com,https://node2.com,https://node3.com
```